### ISSUE TITLE
Why the size of the model which was trained by transfer learning is much less than original model?

### ISSUE THREAD

**Issue by Yaohongchen** [JiahuiYu/generative_inpainting#400]
Hi Jiahui, thanks for your amazing work and open source code. I try transfer learning based on the model which was pre-trained by places2. I use the original model as initialization parameters and the size of data file about 114MB. I retrain the model from allconv11 (the concat of the two branchs) and freeze the parameters of the former layers. However, the size of the final model only 88MB, it make me feel confused.

I change the these code of inpaint_model.py from: 
g_vars = tf.get_collection(
            tf.GraphKeys.TRAINABLE_VARIABLES, 'inpaint_net')
to:
g_vars = tf.get_collection(
            tf.GraphKeys.TRAINABLE_VARIABLES, 'inpaint_net/allconv11/kernel:0')
and run the train.py.

Thank you for your help.

**Comment by JiahuiYu** [comment]
Maybe gradients are also checkpointed? You should be able to figure out by printing every tensor of the saved model.

**Comment by Yaohongchen** [comment]
I will try, thank you for your help.
